app/Models/User.php

```diff
@@ -80,31 +80,31 @@ public function sendPasswordResetNotification($token)
         'password', 'remember_token',
     ];
 
-    public function games(){
+    public function games() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->